client/me/help/controller.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
 		analytics.pageView.record( basePath, 'Help' );
 
 		renderWithReduxStore(
-			React.createElement( Help ),
+			<Help isCoursesEnabled={ config.isEnabled( 'help/courses' ) }/>,
```

Comment:
I added another prop based on this called `showCourseTeaser` in @cd198cf2803d8e020c05626dd6c011105d4c60a3

Eventually it will be composed with more business logic.